src/helper_bitcoin.py

```diff
@@ -1,10 +1,18 @@
+"""
+src/helper_bitcoin.py
+=====================
+"""
+
 import hashlib
+
 from pyelliptic import arithmetic
 
-# This function expects that pubkey begin with \x04
+
 def calculateBitcoinAddressFromPubkey(pubkey):
+    """This function expects that pubkey begin with \x04"""
     if len(pubkey) != 65:
-        print 'Could not calculate Bitcoin address from pubkey because function was passed a pubkey that was', len(pubkey), 'bytes long rather than 65.'
+        print 'Could not calculate Bitcoin address from pubkey because function was passed a pubkey that was', len(
+            pubkey), 'bytes long rather than 65.'
```

Comment:
Maybe use print function or logger?

Reply:
```
>> len("        print 'Could not calculate Bitcoin address from pubkey because function was passed a pubkey that was', len(")
>> 115
```

@PeterSurda you'll never remove the [max-line-length](https://github.com/Bitmessage/PyBitmessage/blob/v0.6/setup.cfg#L10) with such attitude

Reply:
@lakshyacis change into logger and shorten lines.